tests/unit/src/unit/issues/Issue6278.hx

```diff
@@ -0,0 +1,16 @@
+package unit.issues;
```

Comment:
Ok, added a `TestFile.hx`. Also made the test a bit more thorough (not just checking for the absence of an exception, but also that the content was actually overwritten). Let's see if all other targets can handle that...